core/src/main/java/org/openstack4j/model/compute/actions/RebuildOptions.java

```diff
@@ -40,6 +48,23 @@ public RebuildOptions name(String name) {
         add(Option.NAME, name);
         return this;
     }
+    
+    public RebuildOptions userData(String userData) {
+        add(Option.USER_DATA, userData);
+        return this;
+    }
+    
+    public void addPersonality(String path, String contents) {
```

Comment:
I suggest changing the semantics here to `setPersonality`. It will be more aligned with the class api, and it would not need the field.

Reply:
I changed `addPersonality` to `personality` and `addMetadata` to `metadata` like existing other method names.